lib/vanagon/driver.rb

```diff
@@ -144,7 +144,7 @@ def run # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
       @engine.ship_workdir(workdir)
       @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make} #{make_target})")
       @engine.retrieve_built_artifact(@project.artifacts_to_fetch, @project.no_packaging)
-      @project.publish_yaml_settings(@platform)
+      @project.publish_yaml_settings(workdir, @platform)
```

Comment:
During the build is the workdir 'output' or a directory that gets synced back to output?

Reply:
Hmm, it looks like workdir is a temporary directory that's synced back to output, but now that I'm looking more at it, it seems like it'd be better to just use the `platform.output_dir` or something -- update coming soon.